src/shared/App.tsx

```diff
@@ -1,18 +1,19 @@
 import * as React from "react";
 import {Switch} from "react-router-dom";
-import {SingleRoute} from "./components/atoms";
-import {NoMatch} from "./components/pages";
+import {Route} from "./components/atoms";
```

Comment:
I know this file is in `shared`, but should these be updated to use `@shared/atoms` that way if it's ever moved it continues to work? If yes, we should update all of them.

Reply:
Yeah, I'm down to update them. Just didn't want to update all at once. We can follow up in another PR?